### ISSUE TITLE
Documentation of FixedHeatFlow and PrescribedHeatFlow

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#949]
**Reported by rfranke on 3 Jan 2013 09:38 UTC**
The documentation of the models 
```
Modelica.Thermal.HeatTransfer.Sources.FixedHeatTransfer
Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatTransfer
```
says:

**_If parameter alpha is > 0**, the heat flow is mulitplied by (1 + alpha\*(port.T - T_ref)) in order to simulate temperature dependent losses (which are given an reference temperature T_ref)._ 

The implementation 
```
port.Q_flow = -Q_flow*(1 + alpha*(port.T - T_ref));
```
does the multiplication for `alpha <= 0` as well. A negative alpha is even needed, in order to achieve a temperature driven heat transfer between port.T and T_ref.

See for example
```
Modelica.Fluid.Examples.BranchingDynamicPipes
```
using a small alpha to stabilize steady-state initialization or
```
Modelica.Fluid.Examples.HeatingSystem
```
using a large alpha to implement a simple temperature control.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/949

**Comment by modelica-trac-importer** [comment]
**Comment by ahaumer on 7 Jan 2013 10:30 UTC**
Replying to [ticket:949 rfranke]:
Dear Rüdiger,
you're right, the documentation is not correct. However, alpha = 0 leads to a temperature independent heat flow. 
I don't understand your statement "A negative alpha is even needed, in order to achieve a temperature driven heat transfer between port.T and T_ref." The temperature dependency of the heat flow (dependent on the temperature difference between port.T and T_ref) is only switched off for alpha = 0. In general, alpha < 0 could be meaningful - think of electrical resistor materials with negative temperature coefficient. 
What do you suggest to fix? IMHO it is sufficient to correct the documentation.
Best regards, Toni

**Comment by modelica-trac-importer** [comment]
**Comment by rfranke on 7 Jan 2013 14:21 UTC**
A possible fix is to replace

  _If parameter alpha is > 0, ..._

with

  _If parameter alpha is <> 0, ..._

in order to include negative values as well. Or with two sentences:

  _The heat flow is multiplied by (1 + alpha\*(port.T - T_ref)). A parameter alpha < 0 allows to simulate losses to the ambient, depending on the reference temperature T_ref._

**Comment by modelica-trac-importer** [comment]
**Comment by ahaumer on 9 Jan 2013 12:49 UTC**
Ok I'll fix the documentation:
If parameter alpha is <> 0, the heat flow is mulitplied by (1 + alpha\*(port.T - T_ref)) in order to simulate temperature dependent losses (which are given with respect to reference temperature T_ref).

**Comment by modelica-trac-importer** [comment]
**Comment by hansolsson on 9 Jan 2013 14:11 UTC**
Oh, and just to be sure that it is not missed: please correct the spelling as well. Reading the ticket above was painful

mulitplied -> multiplied

**Comment by modelica-trac-importer** [comment]
**Comment by ahaumer on 10 Jan 2013 09:03 UTC**
fixed by 5155b518790818faab1eeec058fa07025f698b1c
